packages/legend-graph/src/graphManager/protocol/pure/v1/V1_PureGraphManager.ts

```diff
@@ -769,6 +775,59 @@ export class V1_PureGraphManager extends AbstractPureGraphManager {
         options,
       );
 
+      // Resovle function's functionName and parameter/returnValu's type/multiplicty.
```

Comment:
// Resolve function's functionName and parameter/returnValue's type/multiplicity